portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/deployment/AutoDeploy.testcase

```diff
@@ -10,6 +10,7 @@ definition {
 	test FragmentAutoDeploy {
 		property skip.start.app.server = "true";
 		property test.name.skip.portal.instance = "AutoDeploy#FragmentAutoDeploy";
+		property testcase.url = "file:///opt/dev/projects/github/liferay-portal/";
```

Comment:
Is it possible to build this into the skip.start.app.server startup in build-test.xml? Since if we are using skip.start.app.server we can assume that its expected that the connection issue can be present for those tests as well